what-is-software-development/what-is-software.md

```diff
@@ -0,0 +1,190 @@
+- Who can't access the software?
+- Who can't navigate the software?
+
+As of 2020, there are no absolute answers to how to build "perfect" software. However, the tech industry is attempting to have these conversations. Being informed on how software has impact on people and the world is crucial to advance these conversations.
+
+### !callout-info
+
+## Code of Conduct
+
+It's valuable to know that a lot of software and developer communities have adopted a code of conduct, or an agreement and commitment to community health. A popular code of conduct for software is based on [Contributor Covenant](https://www.contributor-covenant.org/). Python's code of conduct is also [available online](https://www.python.org/psf/conduct/).
+
+### !end-callout
+
+## Hello, World!
+
+A "Hello, World!" program is the name of a program whose responsibility is to print `Hello, World!`.
+
+The program is small and simple on purpose. "Hello, World!" programs are a common way to introduce a programming language. A "Hello, World!" program should prove to the programmer that the program runs, and they understand some basic syntax.
+
+It's tradition to write a "Hello, World!" program when starting to learn new language!
+
+In Python, we could open up a Python REPL on our local machine, and execute
+
+```python
+print("Hello, World!")
+```
+
+### !callout-info
+
+## You are the Local Machine
+
+What does "local machine" mean? The phrase "local machine" will be used to mean your own very computer that you're writing code on. "Local" refers to a computer close to you, the user. We say "machine" to not assume that the users of code are laptops or desktop computers!
+
+### !end-callout
+
+## Check for Understanding
+
+<!-- Question 1 -->
+
+<!-- prettier-ignore-start -->
+### !challenge
+
+* type: multiple-choice
+* id: 84072fe1-1d71-4ca6-b2b5-e602cee63d2d
+* title: What is Software?
+
+##### !question
+
+Application software is...
+
+##### !end-question
+
+##### !options
+
+* Software that builds computer system functionality
+* Software that users apply to make their lives better
+* Software that typically gets distributed online, such as the App Store
+* Software built for an end user
+
+##### !end-options
+
+##### !answer
+
+* Software built for an end user
+
+##### !end-answer
+
+### !end-challenge
+<!-- prettier-ignore-end -->
+
+<!-- Question 2 -->
+
+<!-- prettier-ignore-start -->
+### !challenge
+
+* type: multiple-choice
+* id: 88476d2f-e03f-4be4-a15c-631a9f9f2a21
+* title: What is Software?
+
+##### !question
+
+Open Source Software means...
+
+##### !end-question
+
+##### !options
+
+* the source code is available to look at and use
+* the source code is built only by volunteers
+* the source code must only be used to build free software
+* the source code is available through the Open Source Foundation
+
+##### !end-options
+
+##### !answer
+
+* the source code is available to look at and use
+
+##### !end-answer
+
+### !end-challenge
+<!-- prettier-ignore-end -->
```

Comment:
This was great. I really like the questions spread throughout the lesson. I didn't see learning goals for this lesson, was that something we wanted to add?

Reply:
Goood point 😆  Thanks, I'll add them after merge